### ISSUE TITLE
Plugin GUI not influencing plugin state

### ISSUE THREAD

**Issue by Spunge** [OpenMusicKontrollers/synthpod#39]
When i use x42-comp dynamic stereo compressor, turning the knobs in the GUI does not influence the plugins parameters. Also, changing the plugins parameters does not influence the GUI, it seems they are not linked.

I'm also not seeing the input signal visualized in the GUI as i do in other hosts.

synthpod UI gives this output:
```
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
```

**Comment by ventosus** [comment]
buffer overflows should be fixed in 5fbcf021517ecdaa4f9b4035eb006beb51820ee8

**Comment by Spunge** [comment]
Nice! 

There's only one part of the x42-comp not working now, and thats the bottom indicator:
![image](https://user-images.githubusercontent.com/1842108/62114308-b27f7400-b2b6-11e9-8de8-638206385aba.png)

This indicator also shows in the flowgraph, where it's also not working:
![image](https://user-images.githubusercontent.com/1842108/62114350-c925cb00-b2b6-11e9-8696-00fbbd37704d.png)

Also, when adding the x42-comp, the indicator in the flowchart will not show, but after a restart it will.

**Comment by Spunge** [comment]
I'm still getting a lot of:
```
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
(UI)  [Error] _mod_uis_send: buffer overflow
```

For example when opening the GUI of x42-whirl-speaker

**Comment by ventosus** [comment]
> For example when opening the GUI of x42-whirl-speaker

Not to worry. x42-whirl-speaker has a lot of changes on its output ports, so the ringbuffer between DSP<->UI is quickly filled up as long as the UI is not yet up and running.

Note to self: subscribe to ports only after UI is fully up and running, not when starting it.

**Comment by ventosus** [comment]
fixed in 550c91803e18de6ba8ce3c48b5ce67d33d6f4d15

**Comment by ventosus** [comment]
> Also, when adding the x42-comp, the indicator in the flowchart will not show, but after a restart it will.

fixed in dd64b3779b75a1bcffea5558b8ed74c2cbbf9e32

**Comment by ventosus** [comment]
> There's only one part of the x42-comp not working now, and thats the bottom indicator:

There's nothing I can fix here. It's the plugin's job to draw all that stuff.
